core/src/com/projectkorra/projectkorra/airbending/AirShield.java

```diff
@@ -125,9 +126,26 @@ public void progress() {
 			this.remove();
 			return;
 		}
+		this.extinguishBlocks();
 		this.rotateShield();
 	}
 
+	private void extinguishBlocks() {
+		for (final Block testblock : GeneralMethods.getBlocksAroundPoint(this.player.getLocation(), this.radius)) {
+			if (FireAbility.isFire(testblock.getType())) {
+				testblock.setType(Material.AIR);
+				testblock.getWorld().playEffect(testblock.getLocation(), Effect.EXTINGUISH, 0);
+			} else if (testblock.getBlockData().getAsString().contains("candle") && testblock.getBlockData().getAsString().contains("lit=true")) {
```

Comment:
For this, rather than checking the string, I think it's better if you check if the material is within the `CANDLES` block tag. This tag contains all candles, and also allows for a more accurate representation of candles when users are running Arclight (modded).

You'll likely need to check if the tag exists, since it doesn't exist in 1.16

Finally, since candles implement the [Lightable](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/block/data/Lightable.html) interface (https://hub.spigotmc.org/javadocs/spigot/org/bukkit/block/data/type/Candle.html), you can set the blockstate without needing to do a string replacement (which will also be faster than a string replacement)